### PR TITLE
[Estuary] Add episodename to live tv and recordings osd sub label, cleanup seek bar for live tv.

### PR DESCRIPTION
**Pull request by ksooo**
Add episodename to live tv and recordings osd sub label, because it was nowhere displayed in the osd:

Before:
screenshot001

After:
screenshot000

Remove show title and episodename from to live tv seek bar dialog, because this info was duplicated in the osd sub label (show title even before this PR):

Before:
screenshot003

After:
screenshot002

@phil65 @ronie mind taking a look.

### CODE REVIEW

File: addons/skin.estuary/xml/DialogSeekBar.xml [hunk 1]
```diff
@@ -209,21 +209,11 @@
...
-			<control type="label">
-				<left>260</left>
-				<right>50</right>
-				<top>20</top>
-				<height>25</height>
-				<label>$INFO[VideoPlayer.Title]$INFO[VideoPlayer.EpisodeName, (,)]</label>
-				<align>left</align>
-				<font>font36_title</font>
-				<aligny>center</aligny>
-			</control>
```
Comment (phil65) [inline addons/skin.estuary/xml/DialogSeekBar.xml]: youre sure this doesnt cause regressions for TV episodes?

Comment (ksooo) [inline addons/skin.estuary/xml/DialogSeekBar.xml]: Yep I'm as sure as one can be. Runtime tested carefully, no regressions found.

Do you have a special scenario in mind?

Comment (ksooo) [inline addons/skin.estuary/xml/DialogSeekBar.xml]: I added episodename to topbar with this PR and show title was displayed in top bar and seek her. Take a look at the screen shots, please.

Comment (phil65) [inline addons/skin.estuary/xml/DialogSeekBar.xml]: ok, just wanted to make sure. :)

**Review by phil65 — APPROVED** [review]

**Comment by ksooo** [comment]
jenkins build this please

**Comment by ksooo** [comment]
jenkins error is unrelated

**Pull request merged**